### PR TITLE
Add deprecated claim names

### PR DESCRIPTION
**Pull request by OR13**
We should keep them in this spec with an issue marker until IESG updates the JWT registry.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/pull/124.html" title="Last updated on Jul 13, 2023, 6:37 PM UTC (9847170)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/124/227499a...9847170.html" title="Last updated on Jul 13, 2023, 6:37 PM UTC (9847170)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -829,6 +831,70 @@ <h2><code>application/vc+ld+sd-jwt</code></h2>
...
   </section>
+  
+
+  <section id="json-web-token-claim-registrations">
+    <h3>JSON Web Token Claims</h3>
+    <p>
+      This section retained deprecated registered claim names, 
+      that have been submitted to the Internet Engineering Steering Group
+      (IESG) for review, approval, and registration with IANA in the
+      "JSON Web Token Claims Registry".
```
Comment (TallTed) [inline index.html:842]: Shouldn't "JSON Web Token Claims Registry" link to that resource, whether via a `[[???]]` reference or a direct `<a href...>`?
```suggestion
      This section retains deprecated registered claim names, 
      that were previously submitted to the Internet Engineering Steering
      Group (IESG) for review, approval, and registration with IANA in the
      "JSON Web Token Claims Registry".
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html:842]: Yes, I wonder why nobody did that in VCDM?

I prefer not to make a bunch of changes here, until we finish moving the section, from there to here... but I agree that everything should be linked that can be... we should save that for editorial though.

Comment (TallTed) [inline index.html:842]: Fine, save the link addition for later. But these text tweaks (relocated from above) are focused on your "Add deprecated claim names" changes.
```suggestion
      This section retains deprecated registered claim names 
      that were previously submitted to the Internet Engineering Steering
      Group (IESG) for review, approval, and registration with IANA in the
      "JSON Web Token Claims Registry".
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html:842]: I'm good to merge these as is when https://github.com/w3c/vc-data-model/pull/1195 is merged

Comment (OR13) [reply index.html:842]: I guess I can take them now if there are no objections.

Comment (OR13) [reply index.html]: I merged them anyway, we will still need to wait for https://github.com/w3c/vc-data-model/pull/1195 to merge

**Comment by OR13** [comment]
Waiting for https://github.com/w3c/vc-data-model/pull/1195 to merge before this.

**Review by selfissued — APPROVED** [review]
Makes sense to me

**Review by mprorock — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Pull request merged**